package.json

```diff
@@ -29,11 +29,10 @@
   "homepage": "https://github.com/CanopyTax/node-jspm-jasmine#readme",
   "devDependencies": {
     "babel-cli": "^6.6.5",
-    "babel-preset-es2015": "^6.6.0"
-  },
-  "peerDependencies": {
-    "jasmine": "*",
-    "jspm": "*"
+    "babel-preset-es2015": "^6.6.0",
+    "commander": "^2.9.0",
+    "jasmine": "^2.4.1",
+    "jspm": "^0.17.0-beta.12"
```

Comment:
I misunderstood the "deprecation" of peerDependencies in npm3, it does make a lot of sense to leave jspm as a peerDependency. I'll leave it to you to decide what's best for jasmine (I would probably leave it as a peerDependency since chances are users are already using jasmine for frontend tests and it would be nice to use the same jasmine version throughout). On this note, I overlooked the fact that we're not bundling, so "commander" has to be a true dependency. Let me know what you would like to do.